NOTICE.txt

```diff
@@ -159,8 +159,8 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 --------------------------------------------------------------------
 Dependency: github.com/aws/aws-sdk-go-v2
-Version: v2.0.0-preview.5
-Revision: d52522b5f4b95591ff6528d7c54923951aadf099
+Version: v0.9.0
```

Comment:
@andrewvc We are curious how the NOTICE.txt get updated 😬 Is this a part of the mage/make command? cc @sayden

Reply:
`make update` in the root does this.

Reply:
Thank you!